test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -671,6 +691,52 @@ describe('33acrossBidAdapter:', function () {
         const [ buildRequest ] = spec.buildRequests(bidRequests);
         validateBuiltServerRequest(buildRequest, serverRequest);
       });
+
+      it('returns the viewport dimensions of the top most accessible frame', function() {
```

Comment:
does this test the case when top-level element is indeed accessible?

Reply:
I think this unit test could be enough for testing the algorithm that obtains the top most accesible window.

But at the same time it might NOT be that clear that we are covering the other scenario (all accessible windows), I've added another commit with some changes that makes a little bit more clear what we are testing.